### PR TITLE
V2 Pages Build image

### PR DESCRIPTION
**Pull request by GregBrimble**
image

### CODE REVIEW

File: content/pages/_partials/_platform-language-support-and-tools/v1.md [hunk 1]
```diff
@@ -11,9 +11,6 @@ languages:
   - name: Elixir
     default: "1.7"
     supported: "1.7 only"
-  - name: Emacs
```
Comment (WalshyDev) [inline content/pages/_partials/_platform-language-support-and-tools/v1.md:14]: ?

**Review by WalshyDev — APPROVED** [review]

File: content/pages/_partials/_platform-language-support-and-tools/v1.md (hunk 1 above)
Comment (GregBrimble) [reply content/pages/_partials/_platform-language-support-and-tools/v1.md:14]: It was already listed in the tools section for v1 which felt like a better home 🏠

File: content/pages/platform/language-support-and-tools.md [hunk 2]
```diff
@@ -14,6 +14,20 @@ Cloudflare Pages' build environment has broad support for a variety of languages
 
 If you need to use a specific version of a language, (for example, Node.js or Ruby) you can specify it by providing an associated environment variable in your build configuration, or setting the relevant file in your source code.
 
+{{<Aside type="warning" header="v2 Beta">}}
+
+We've recently annouced a [v2 beta](https://blog.cloudflare.com/moderizing-cloudflare-pages-builds-toolbox/) which brings several improvements to our build system. To migrate to this new version, configure your Pages project settings in the dashboard. Go to **Settings** > **Build & deployments** > **Build system version** and select the latest version.
```
Comment (deadlypants1973) [inline content/pages/platform/language-support-and-tools.md]:
```suggestion
We have recently annouced a [v2 beta](https://blog.cloudflare.com/moderizing-cloudflare-pages-builds-toolbox/) which brings several improvements to our build system. To migrate to this new version, configure your Pages project settings in the dashboard:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account. 
2. Select **Workers & Pages** and select your Pages project.
3. Go to **Settings** > **Build & deployments** > **Build system version** and select the latest version.
```

File: content/pages/platform/language-support-and-tools.md [hunk 3]
```diff
@@ -14,6 +14,20 @@ Cloudflare Pages' build environment has broad support for a variety of languages
...
+
+Notably, we've made changes to the default versions of languages and tools that are included. We welcome feedback on v2 beta's selection in our [Discord server](https://discord.com/invite/cloudflaredev): if there are tools and languages you use that are missing, or if you spot any problems when migrating to v2, please let us know!
```
Comment (deadlypants1973) [inline content/pages/platform/language-support-and-tools.md]:
```suggestion
Notably, we have made changes to the default versions of languages and tools that are included. Submit feedback on v2 beta's selection in the [Cloudflare Developer Discord](https://discord.com/invite/cloudflaredev). If there are tools and languages you use that are missing, or if you experience any problems when migrating to v2, submit your feedback in the [Cloudflare Developer Discord](https://discord.com/invite/cloudflaredev).
```

File: content/pages/platform/language-support-and-tools.md [hunk 4]
```diff
@@ -14,6 +14,20 @@ Cloudflare Pages' build environment has broad support for a variety of languages
...
+
+If you were previously relying on the default versions of any languages or tools in the build system, your build may fail when migrating to v2. To fix this, you must specify the specific version you wish to use. Details on how to do this for each of these languages and tools are further down on this page. For example, if you were previously relying on the default version of Node.js in the v1 build system, to migrate to v2, you must specify that you need Node.js 12.18.0 by setting an `NODE_VERSION` environment variable or by adding a `.node-version` or `.nvmrc` file to your project.
+
+During this beta period, we are aware of, and are in the process of fixing, the following issues:
+
+- `Pipfile` support for setting a Python version
```
Comment (deadlypants1973) [inline content/pages/platform/language-support-and-tools.md]:
```suggestion
If you were previously relying on the default versions of any languages or tools in the build system, your build may fail when migrating to v2. To fix this, you must specify the version you wish to use. Find details on how to do this for each of these languages and tools on this page. For example, if you were previously relying on the default version of Node.js in the v1 build system, to migrate to v2, you must specify that you need Node.js `12.18.0` by setting a `NODE_VERSION` environment variable or by adding a `.node-version` or `.nvmrc` file to your project.

During this beta period, we are aware of, and are in the process of fixing, the following issues:

- `Pipfile` support for setting a Python version
```

**Review by deadlypants1973 — APPROVED** [review]
PCX review

**Pull request merged**